cluster/gce/gci/configure-helper.sh

```diff
@@ -2111,9 +2112,15 @@ function start-kube-scheduler {
   # Scheduler Component Config takes precedence over some flags.
   if [[ -n "${KUBE_SCHEDULER_CONFIG:-}" ]]; then
     create-kube-scheduler-config
+    local config_path
+    config_path=$(awk '/kubeconfig:/{print $2;}' /etc/srv/kubernetes/kube-scheduler/config)
```

Comment:
rather than pulling out the kubeconfig file, should we just use the one created above in `create-kubeconfig "kube-scheduler" "${KUBE_SCHEDULER_TOKEN}"` and move the authentication-kubeconfig/authorization-kubeconfig flags outside of the KUBE_SCHEDULER_CONFIG if/else blocks?

Reply:
wow thanks for pointing it out 👍 , I should not have missed that. I have updated the script as you said, PTAL.